lupa/_lupa.pyx

```diff
@@ -1267,8 +1267,14 @@ cdef object execute_lua_call(LuaRuntime runtime, lua_State *L, Py_ssize_t nargs)
     # call into Lua
     with nogil:
         result_status = lua.lua_pcall(L, nargs, lua.LUA_MULTRET, 0)
-    #runtime.reraise_on_exception()
     if result_status:
+        python_errors = (
+            'error reading Python attribute/item',
+            'error during Python call',
+        )
```

Comment:
Looking at error message texts seems like a hack to me. Isn't there a better way to do what you want to achieve here?